### ISSUE TITLE
Dark Mode / Adjust Style to Editor and Studio

### ISSUE THREAD

**Issue by marwyg** [opencast/opencast-admin-interface#1265]
Would be cool to have a dark mode. I also heard from other, that it would be generally nice to adapt the other styles from Studio and the Editor, so everything would be consistent.

I played a little bit with the developer tools and took a screenshot to give everyone some eye candy :D

"Hmmm nice.. so dark.. so cool"
![Image](https://github.com/user-attachments/assets/ef104ec0-2ab0-4fb5-ad9c-baa7e527e2ea)

(Note: the colors should be the same as in Studio.. I used the css revert feature for some parts, because of this the links are orange and the delete button blue. That is of course not the intended "end result" 😂)

**Comment by Arnei** [comment]
I think we should clearly differentiate between just adding dark mode, or aligning the admin ui with Studio/Editor. The latter is way more coding work and also would require a lot more design work (though it would also give us dark mode "for free").

**Comment by KatrinIhler** [comment]
Closing this in favor of #1293
